libs/events/README.md

```diff
@@ -107,7 +107,7 @@ type EventData interface {
 }
 ```
 
-Generic event data can be typed and registered with tendermint/go-amino
+Generic event data can be typed and registered with [tendermint/go-amino](https://github.com/tendermint/go-amino)
```

Comment:
Do we delete this entirely? I think that was the consensus for main?

Reply:
I would remove it too.

Reply:
https://github.com/cometbft/cometbft/pull/224#discussion_r1093436014